testing/main.lua

```diff
@@ -0,0 +1,216 @@
+-- & 'c:\Program Files\LOVE\love.exe' ./ --console 
+-- /Applications/love_12.app/Contents/MacOS/love ./testing
+
+-- load test objs
+require('classes.TestSuite')
+require('classes.TestModule')
+require('classes.TestMethod')
+
+-- create testsuite obj
+love.test = TestSuite:new()
+
+-- load test scripts if module is active
+if love.audio ~= nil then require('tests.audio') end
+if love.data ~= nil then require('tests.data') end
+if love.event ~= nil then require('tests.event') end
+if love.filesystem ~= nil then require('tests.filesystem') end
+if love.font ~= nil then require('tests.font') end
+if love.graphics ~= nil then require('tests.graphics') end
+if love.image ~= nil then require('tests.image') end
+if love.math ~= nil then require('tests.math') end
+if love.physics ~= nil then require('tests.physics') end
+if love.sound ~= nil then require('tests.sound') end
+if love.system ~= nil then require('tests.system') end
+if love.thread ~= nil then require('tests.thread') end
+if love.timer ~= nil then require('tests.timer') end
+if love.video ~= nil then require('tests.video') end
+if love.window ~= nil then require('tests.window') end
+
+-- love.load
+-- load given arguments and run the test suite
+love.load = function(args)
+
+  -- setup basic img to display
+  if love.window ~= nil then
+    love.window.setMode(360, 240, {
+      fullscreen = false,
+      resizable = true,
+      centered = true
+    })
+    if love.graphics ~= nil then
+      love.graphics.setDefaultFilter("nearest", "nearest")
+      love.graphics.setLineStyle('rough')
+      love.graphics.setLineWidth(1)
+      Logo = {
+        texture = love.graphics.newImage('resources/love.png'),
+        img = nil
+      }
+      Logo.img = love.graphics.newQuad(0, 0, 64, 64, Logo.texture)
+      Font = love.graphics.newFont('resources/font.ttf', 8, 'normal')
+      local txtobj = love.graphics.newTextBatch or love.graphics.newText
+      TextCommand = txtobj(Font, 'Loading...')

```

Comment:
I think these can just be `love.graphics.print` calls inside love.draw instead of needing TextBatch objects